get_team_roles/team_roles.py

```diff
@@ -1,39 +1,64 @@
 import argparse
 import sys
 import pdpyras
+import csv
 
-def get_teams(session, comma_separated):
-    if comma_separated:
-        sys.stdout.write("Team ID, Team Name, User ID, User name, Team role\n")
+def get_teams(session):
+    if args.comma_separated:
+        sys.stdout.write("Team ID', 'Team Name', 'User ID', 'User name', 'Email', 'Base role', 'Team role'\n")
```

Comment:
missing a `'` before `Team ID`

Reply:
also, probably worth changing all these `sys.stdout.write`s to `print()` statements. [There's very little difference](https://stackoverflow.com/questions/3263672/the-difference-between-sys-stdout-write-and-print), and it will make it a bit more readable/obvious. It will also allow you to remove the `\n` from the end of each string.